Sources/TerminalAPIKit/Models/Operation.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Operation.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public final class Operation: Codable {
+
+    /// The type of request after which the waiting screen shows: InputRequest, DisplayRequest, and Payment
```

Comment:
we probably don't need to mention the options here, since the type is now an `enum`, and the possible values will be restricted to that.

Reply:
Latest commit should handle this